valhalla/app/javascript/mixins/manifesto-filemanager-mixins.js

```diff
@@ -0,0 +1,80 @@
+const ManifestoFilemanagerMixins = {
+
+  mainSequence: function () {
+    // mainSequence is the one without an id (not ideal since it could have an id)
```

Comment:
I'm not sure how to read this comment.  When would it have an id? Is this an unresolved issue where we need some other way to identify it?

Reply:
Sorry, this an old note to self. However, we need to determine which is the default sequence if there are multiple. The spec says it would be the first in the list, so I will just note that and remove the cruft.